src/metax_api/migrations/0022_auto_20200527_1639.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.9 on 2020-05-27 13:39
```

Comment:
Is this file relevant in this ticket?

Reply:
It's no, some how got here after testing another pr.